test/kubemark/stop-kubemark.sh

```diff
@@ -24,6 +24,7 @@ source "${KUBE_ROOT}/cluster/kube-env.sh"
 detect-project &> /dev/null
 
 MASTER_NAME="${INSTANCE_PREFIX}-kubemark-master"
+EVENT_STORE_NAME="${INSTANCE_PREFIX}-event-store"
```

Comment:
This is a duplicate of what's in start-kubelet.sh.  Can we factor it out into a common place?